### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by Elhambasir**
### What is done in this PR?

Added missing JSX elements to the project (without adding extra functionality) and implemented all styles based on Zeplin's design (this includes all elements + the same styling).

Note: If you are using a small screen (Laptop) to open this project you need to zoom out the browser window to see the complete design because this project is not responsive and maybe does not fit the small screen.

Thank you for reviewing.

### CODE REVIEW

File: src/styles/renderIndividual.module.css [hunk 1]
```diff
@@ -18,7 +18,56 @@
   display: flex;
   justify-content: center;
   align-items: center;
-  gap: 20px;
+  gap: 10px;
+}
+
```
Comment (tresorsawasawa) [inline src/styles/renderIndividual.module.css]: - [x] As required for the project, your design to stick as much as possible to the one provided on [Zeplin](https://zeplin.io/) and All elements in design to be added. kindly ensure that you improve your `Header` styiling by adding the missing element, using the correct fonts(size, weight, family,...) on text, correct alignment, padding, margin, text color, Uppercase,... (Here sis the blue color to be used:` #0290ff`)
### Template design 
Screenshot from 2023-05-19 18-32-47
### Your design 
Screenshot from 2023-05-19 18-33-23

File: src/styles/renderIndividual.module.css [hunk 2]
```diff
@@ -18,7 +18,56 @@
...
+  stroke-dashoffset: 0;
+  stroke: #03a9f4;
+}
+
+.rectangle3 svg circle:nth-child(2) {
+  stroke-dashoffset: -99;
+  stroke: #f3f3f3;
 }
 
 .renderWrapperCurrentChapter {
```
Comment (tresorsawasawa) [inline src/styles/renderIndividual.module.css]: As required for the project, your design to stick as much as possible to the one provided on [Zeplin](https://zeplin.io/) and All elements in design to be added. kindly ensure that you improve your Header styling by using the correct fonts(size, weight, family,...) on text, correct alignment, padding, margin, text color, height, background color on buttons, spacing between book item, height of book item,... Just see the below images to see the difference
#### Template design
Screenshot from 2023-05-19 18-48-53
#### Your design
Screenshot from 2023-05-19 18-47-56

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Elhambasir 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow Gitflow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles/renderIndividual.module.css (hunk 1 above)
Comment (Elhambasir) [reply src/styles/renderIndividual.module.css]: I got it

File: src/styles/renderIndividual.module.css (hunk 2 above)
Comment (Elhambasir) [reply src/styles/renderIndividual.module.css]: Ok thank you

**Review by KayLemba — APPROVED** [review]
[[# STATUS: APPROVED :green_circle: :clap: 

Hello @Elhambasir       , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉  Keep up the good work 🥂 

HIGHLIGHTS:

- [x]  Your repo is professional  😎 
- [x] Descriptive P.R 💯 
- [x] linters are passing
- [x] Professional README 💯
- [x] Nice UI design💯 
 










 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]]

**Comment by Elhambasir** [comment]
Thank you

**Pull request merged**